bookkeeper-server/src/main/java/org/apache/bookkeeper/client/ReadLastConfirmedOp.java

```diff
@@ -140,17 +149,15 @@ public synchronized void readEntryComplete(final int rc, final long ledgerId, fi
         }
 
         if (numResponsesPending == 0 && !completed) {
-            int totalExepctedResponse = lh.getLedgerMetadata().getWriteQuorumSize()
```

Comment:
@rdhabalia PTAL

Reply:
yes, change looks good, we don't need this logic if `coverageSet.checkCovered()` checks correct quorum response.
However, as I explained in another comment: I think `coverageSet.checkCovered()` doesn't return true if `E=3, W=2, A=2` and if client receives ack from one of the bookie.